doc_user/UserManual.adoc

```diff
@@ -1975,15 +1980,14 @@ The patient is already selected, while other fields need to be set:
 * *Ward*: to specify the ward where the patient has to be received (it will be shown in the Worksheet, see
 <<worksheet,Worksheet>>)
 * *Service*: a brief description of the service that will be provided to the patient (optional)
-* *Duration (Min)*: the estimated duration of the service that will be provided (optional)
+* *Duration (Min)*: the estimated duration of the service that will be provided (optional, default is 30min)
```

Comment:
it would be clearer if `default is 30min` was `default is 30 minutes`

Reply:
Done, thanks!